pkgs/applications/networking/instant-messengers/tangram/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitHub, appstream-glib, desktop-file-utils, gdk-pixbuf
+, gettext, gjs, glib, gobject-introspection, gsettings-desktop-schemas, gtk3
+, hicolor-icon-theme, meson, ninja, pkgconfig, python3, webkitgtk, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tangram";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "sonnyp";
+    repo = "Tangram";
+    rev = "v${version}";
+    sha256 = "0bhs9s6c2k06i3cx01h2102lgl7g6vxm3k63jkkhh2bwdpc9kvn3";
+    fetchSubmodules = true;
+  };
+
+  buildInputs =
+    [ gdk-pixbuf gjs glib gsettings-desktop-schemas gtk3 webkitgtk ];
```

Comment:
```suggestion
  buildInputs = [ gdk-pixbuf gjs glib gsettings-desktop-schemas gtk3 webkitgtk ];
```

Reply:
Not a `nixfmt` fan eh?

Reply:
Not a 80 line length fan.

Reply:
Fair enough, is there a style guide somewhere?

Reply:
The contributing guide should have some style guiding but there is not everything documented. I mostly try to go for a uniform style which is easily grep-able but not too nity on everything.